README.md

```diff
@@ -851,7 +847,14 @@ ANSWERS.addComponent('VerticalResults', {
     // Optional, whether to display all results in the vertical when no results are found. Defaults to false, in which case only the no results card will be shown.
     displayAllResults: false
   },
+  // Optional, whether to display the total number of results, default true
+  // DEPRECATED: we recommend setting hideResultsHeader to true, and using the VerticalResultsCount and AppliedFilters components instead.
+  showResultCount: true,
+  // Optional, a custom template for the results count. You can specify the variables resultsCountStart, resultsCountEnd, and resultsCount.
+  // DEPRECATED: we recommend setting hideResultsHeader to true, and using the VerticalResultsCount and AppliedFilters components instead.
+  resultsCountTemplate: '<div>{{resultsCountStart}} - {{resultsCountEnd}} of {{resultsCount}}</div>',
   // Configuration for the applied filters bar in the header.
+  // DEPRECATED: we recommend setting hideResultsHeader to true, and using the VerticalResultsCount and AppliedFilters components instead.
   appliedFilters: {
     // If true, show any applied filters that were applied to the vertical search. Defaults to true
     show: true,
```

Comment:
This is a lot of text in the readme dedicated to deprecated options. What are you thoughts on putting a single message such as // DEPRECATION NOTICE: The options showResultsCount, resultsCountTemplate, and appliedFilters are deprecated. We recommend setting hideResultsHeader to true, and using the VerticalResultsCount and AppliedFilters components instead.

This would let us remove a lot of text including the appliedFilters options. If people need to see the options they can look at an older release

Reply:
yeah I think something like that would make a lot of sense!